molecule/default/tests/test_default.py

```diff
@@ -1,53 +0,0 @@
-"""Molecule tests for the default scenario."""
-
-import os
-
-import testinfra.utils.ansible_runner
-
-
-testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ["MOLECULE_INVENTORY_FILE"]
-).get_hosts("all")
-
-
-def test_nix_config(host):
-    """Test that nix was installed as expected."""
-    config_dir = host.file("/home/molecule/.config")
-    nix_conf = host.file("/home/molecule/.config/nix/nix.conf")
-
-    assert config_dir.is_directory
-    assert config_dir.mode == 0o0755
-    assert config_dir.user == "molecule"
-    assert nix_conf.is_file
-    assert nix_conf.mode == 0o0644
-    assert nix_conf.user == "molecule"
-    assert "nix-command flakes" in nix_conf.content_string
```

Comment:
@nre-ableton AFAICT, this assertion was not converted to `verify.yml`.

Reply:
Good call, I must have overlooked this. See https://github.com/Ableton/ansible-role-nix/pull/85.

Reply:
Thanks!